virtool/validators.py

```diff
@@ -3,6 +3,7 @@
 import virtool.users.utils
 
 RE_HEX_COLOR = re.compile("^#([A-Fa-f0-9]{6}|[A-Fa-f0-9]{3})$")
+RE_EMAIL = re.compile("^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
```

Comment:
Did you review the `email-validator` package?

Reply:
I did, I wasn't sure if you would prefer me to use that or not though